day_7/caesar_cipher.rb

```diff
@@ -0,0 +1,30 @@
+print "what is your word?"
+print "> "
+text = $stdin.gets.chomp.downcase
+print "number of characters to shift"
+print "> "
+number = $stdin.gets.chomp
+
+def caesar_cipher (string, number)
+  string_array = string.split("")
+  number = number.to_i
+  array = []
+  string_array.each do |letter|
+    result = letter.ord - 97
+    num = result.to_i
+    num += number
+    num += 97
+    new_result = num.chr("UTF-8")
+    array.push(new_result)
+  end
+  p array.join
+end
+
+
+
+caesar_cipher(text, number)
```

Comment:
Excellent work on this! Good notes for edge cases to think of with this too. 

Tip and One more edge case: 
- You can simply use `gets` without the `$stdin` and it will still run. 
- What happens when you're shifting letters at the end of the alphabet?